From b6906fc97dd2b34bb2e01b5bf5bd742562ad0132 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Fri, 8 May 2020 11:35:08 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable
 CONFIG_TOUCHSCREEN_HYN_CST2XX and CONFIG_GS_SC7A30

For rk3368a tablet.

Change-Id: I588802e3ee785cd55327c5b5edb7d256b41403ef
Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 34cb0d207dd3..2b6ab3ef95d7 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -431,6 +431,7 @@ CONFIG_TOUCHSCREEN_GSLX680_VR=y
 CONFIG_TOUCHSCREEN_GSL3673=y
 CONFIG_TOUCHSCREEN_GSL3673_800X1280=y
 CONFIG_TOUCHSCREEN_GT9XX=y
+CONFIG_TOUCHSCREEN_HYN_CST2XX=y
 CONFIG_TOUCHSCREEN_GT1X=y
 CONFIG_ROCKCHIP_REMOTECTL=y
 CONFIG_ROCKCHIP_REMOTECTL_PWM=y
@@ -443,6 +444,7 @@ CONFIG_GS_KXTJ9=y
 CONFIG_GS_LIS3DH=y
 CONFIG_GS_MMA7660=y
 CONFIG_GS_MC3230=y
+CONFIG_GS_SC7A30=y
 CONFIG_GS_MXC6655XA=y
 CONFIG_GS_LSM303D=y
 CONFIG_LSM330_ACC=y
-- 
2.35.3

